scripts/run_tests_within_container.sh

```diff
@@ -3,6 +3,7 @@ script_dir=$(dirname "$(readlink -f "$0")")
 export KB_DEPLOYMENT_CONFIG=$script_dir/../deploy.cfg
 export KB_AUTH_TOKEN=`cat /kb/module/work/token`
 export PYTHONPATH=$script_dir/../lib:$PATH:$PYTHONPATH
+export PYTHONPATH=$script_dir/../test:$PATH:$PYTHONPATH
```

Comment:
I realize that this is how it was before, but adding `$PATH` to the `$PYTHONPATH` is really weird. If the tests pass without doing that we should take it out.

Reply:
updated!

Reply:
:+1: